### PR TITLE
docs: fix link to Promtail documentation

### PR DESCRIPTION
**Pull request by Birdi7**
**What this PR does / why we need it**: 

The current link to Promtail installation documentation in the file leads to 404 error. I observed that the `../docs/clients/` folder was refactored into `../docs/sources/clients` at same time in past. However, the `loki/docs/sources/clients/promtail/installation.md` file is not very convenient if we talk about UX. So, I've replaced the wrong link with the one to rendered beautiful documentation

### CODE REVIEW

File: production/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Currently there are five ways to try out Loki, in order from easier to hardest:
 - [Build Loki from source](#build-and-run-from-source)
 - [Get inspired by our production setup](#get-inspired-by-our-production-setup)
 
-For the various ways to run `promtail`, the tailing agent, see our [Promtail documentation](../docs/clients/promtail/README.md).
+For the various ways to run `promtail`, the tailing agent, see our [Promtail documentation](https://grafana.com/docs/loki/latest/clients/promtail/installation/).
```
Comment (dannykopping) [inline production/README.md:11]: Can we stick to the relative pathing instead?

**Review by dannykopping — COMMENTED** [review]
Thanks very much for your contribution @Birdi7!

Left you a comment

File: production/README.md (hunk 1 above)
Comment (Birdi7) [reply production/README.md]: Thank you for your comment! I've replaced the link with the relative path to a file with installation instruction. However, I doubt its quality. Could you please double-check if this is what you meant?
Commit with update — b9c18a19f2a926fd397323ecf941ea4c57a73b0b

**Review by dannykopping — APPROVED** [review]
Thanks a lot @Birdi7!

**Pull request merged**